### PR TITLE
all: make stretch the default

### PR DESCRIPTION
**Pull request by kortschak**
@dlech Please take a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 The goal is to implement a simple Go style ev3dev API and helpers for common tasks.
 
-github.com/ev3go/ev3dev depends on an 18-ev3dev kernel or better (http://www.ev3dev.org/news/2017/01/25/kernel-release-cycle-18/).
+github.com/ev3go/ev3dev depends on ev3dev stretch and has been tested on kernel 4.9.50-ev3dev-1.6.0. For jessie support see the [ev3dev-jessie branch](https://github.com/ev3go/ev3dev/tree/ev3dev-jessie).
```
Comment (dlech) [inline README.md]: The current kernel is 4.14.?-ev3dev-2.2.2, so this testing is a bit behind.

File: lego_port.go [hunk 2]
```diff
@@ -179,7 +179,9 @@ func (p *LegoPort) Uevent() (map[string]string, error) {
 }
 
 // ConnectedTo returns a description of the device attached to p in the form
-// {inX,outY}:DEVICE-NAME, where X is in {1-4} and Y is in {A-D}.
+// PLATFORM-ports:{inX,outY}:DEVICE, where X is in {1-4} and Y is in {A-D},
+// or if on a BrickPi3 platform, spi0.1:{SX,MY}:DEVICE, where X is in {1-16}
+// and Y is in {A-P}.
```
Comment (dlech) [inline lego_port.go]: I'm not sure if you care that the description covers all platforms, but BrickPi an PiStorms don't match the patterns listed here. You could generalize the pattern even further at `{device connection}:{port name}` where device connection is the underlying transport (spi0.1, serial0-0, ev3-ports, evb-ports, pistorms) and port name is the name physically printed on the device (with the "in" and "out" prefix for EV3)

File: README.md (hunk 1 above)
Comment (kortschak) [reply README.md]: I'll see about doing some hardware testing soon (the number here is out of date and I have tested more recently, but I don't recall versions).

File: lego_port.go (hunk 2 above)
Comment (kortschak) [reply lego_port.go]: Done.

**Comment by kortschak** [comment]
PTAL

**Comment by dlech** [comment]
LGTM

**Pull request merged**